gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApiResource.java

```diff
@@ -1041,4 +1062,15 @@ private void checkApiReviewWorkflow(final GenericApiEntity api, final String act
             }
         }
     }
+
+    private static Stream<Excludable> mapExcludedExportParams(@Nullable String ex) {
+        if (ex == null) {
+            return Stream.empty();
+        }
+        var excludable = MAPPING_EXCLUDE_EXPORT_PARAMS.get(ex);
+        if (excludable == null) {
+            log.warn("Unknown excluded '{}'", ex);
```

Comment:
I would use a debug logger in that case. People managing the platform can't do anything to fix that "warn" as it comes from a user input